src/main/java/com/example/demo/api/controllers/UserController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.api.controllers;
+
+import com.beust.ah.A;
+import com.example.demo.api.dto.AnswerDto;
+import com.example.demo.api.dto.UserDto;
+import com.example.demo.api.exception.UserNotFoundException;
+import com.example.demo.api.persistence.dao.UserDao;
+import com.example.demo.api.persistence.entity.UserEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class UserController {
+
+    private static final String USER_TICKETS="api/user";
+    private final UserDao userDao;
+
+    public UserController(UserDao userDao) {
+        this.userDao = userDao;
+    }
+
+    @GetMapping(USER_TICKETS)

```

Comment:
for mapping it's fine to use here string value without separate field variable :)